flutter_app/lib/mavlink_communication.dart

```diff
@@ -6,63 +6,61 @@ import 'package:flutter_libserialport/flutter_libserialport.dart';
 import 'package:dart_mavlink/mavlink.dart';
 import 'package:dart_mavlink/dialects/common.dart';
 
-enum MavlinkCommunicationConnection { tcp, serial }
+enum MavlinkCommunicationType {
+  TCP,
+  SERIAL,
+}
```

Comment:
Why did this change?

Reply:
I thought enums were supposed to be all caps but if not I can change this back

Reply:
`lowerCamelCase` : https://dart.dev/effective-dart/style#prefer-using-lowercamelcase-for-constant-names

Reply:
`lowerCamelCase` : https://dart.dev/effective-dart/style#prefer-using-lowercamelcase-for-constant-names